license/COPYRIGHT_HEADER.txt

```diff
@@ -1,4 +1,4 @@
 /*
- * Copyright 2010-2021 JetBrains s.r.o. and Kotlin Programming Language contributors.
+ * Copyright 2010-2022 JetBrains s.r.o. and Kotlin Programming Language contributors.
```

Comment:
Changing this will cause all generated files (such as generated tests) to change, and we have a test that checks that generated files are up-to-date. So it requires regenerating everything.

Also, we're in the process of merging a large PR right now (https://github.com/JetBrains/kotlin/pull/4560) where new generated code is introduced, so I'd postpone this copyright year change before that PR is merged, to minimize number of conflicts.

Reply:
ok when can I fetch and create a pull request again